docs/content/preview/yugabyte-platform/administer-yugabyte-platform/high-availability.md

```diff
@@ -188,3 +188,7 @@ If you are using custom ports for Prometheus in your YugabyteDB Anywhere install
     The default Prometheus port for YugabyteDB Anywhere is `9090`. Custom ports are configured through the settings section of the Replicated installer UI that is typically available at `https://<yugabyteanywhere-ip>:8800/`.
 
     For information on how to access the Replicated settings page, see [Install YugabyteDB Anywhere](../../install-yugabyte-platform/install-software/default/).
+
+## Troubleshooting
+
+If you face issues setting up a high availability configuration when the YBA instances are setup to use the https protocol, please follow the steps above to add CA certificates appropriately to the trust store. If that does not solve the problem, one workaround is to relax the certificate validation requirements by enabling the runtime configuration `yb.ha.ws.ssl.loose.acceptAnyCertificate` (to set it to `true`).
```

Comment:
```suggestion
If you face issues configuring high availability when the YBA instances are configured to use the HTTPS protocol, attempt the steps mentioned in the preceding sections to add CA certificates appropriately to the trust store. In case the issue persists, consider relaxing the certificate validation requirements as a workaround, by enabling the runtime configuration `yb.ha.ws.ssl.loose.acceptAnyCertificate` (set the flag to `true`).
```